qiskit/transpiler/preset_passmanagers/level1.py

```diff
@@ -179,11 +178,7 @@ def _direction_condition(property_set):
     def _opt_control(property_set):
         return not property_set['depth_fixed_point']
 
-    if basis_gates and ('u1' in basis_gates or 'u2' in basis_gates or
-                        'u3' in basis_gates):
-        _opt = [Optimize1qGates(basis_gates), CXCancellation()]
-    else:
-        _opt = [Optimize1qGatesDecomposition(basis_gates), CXCancellation()]
+    _opt = [Optimize1qGatesDecomposition(basis_gates), CXCancellation()]
```

Comment:
This change looks good, but could you add a release note about this change? It should mostly be invisible to users, but it's a bit change for anyone out there still using u1, u2, u3 with a performance penalty so it'll be good to highlight in the release notes.

Reply:
Will do.

Just a note. We were getting about a 10% failure on test_isometry. We added a seed to fix it in the short-term. It appears that if there is a U2 in the transpiled circuit in the test, the isometry from the circuit is out of tolerance by 1 or 2 decimal places from the random_unitary(16) call. U1 was not an issue. I plan to open an issue on that once this is complete.

Reply:
@mtreinish @kdk Found the test_isometry bug and its fixed. Added release note.